### ISSUE TITLE
Does it support cross-platform?

### ISSUE THREAD

**Issue by xlxhn** [subzeroid/instagrapi#1767]
windows and linux

**Comment by subzeroid** [comment]
Yes. `instagrapi` supports Python 3.10+ and is not limited to one desktop OS.

Install the latest release with:

```bash
python -m pip install --upgrade instagrapi
```

References:
- README: https://github.com/subzeroid/instagrapi/blob/master/README.md#L23
- Getting started: https://subzeroid.github.io/instagrapi/getting-started
- PyPI: https://pypi.org/project/instagrapi/

Closing this as answered.
